Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/parser/CommentParser.java

```diff
@@ -243,9 +246,30 @@ public CharSequence handleDead(
             Theme theme, PostParser.Callback callback, Post.Builder builder, CharSequence text, Element deadlink
     ) {
         // html looks like <span class="deadlink">&gt;&gt;number</span>
-        int postNo = Integer.parseInt(Parser.unescapeEntities(deadlink.text(), true).substring(2));
+        // Or like this &gt;&gt;&gt;/int/116000000 in case of a dead crossboard link
+
+        String deadlinkText = deadlink.text();
+        int postNo = -1;
+
+        try {
+            // Handle the crossboard dead link first, because otherwise the condition will be always
+            // true. The order of conditions matters!
+            if (deadlinkText.startsWith(">>>")) {
```

Comment:
This won't work at all. There's a reason why I unescape entities first before doing anything else, because `&gt;&gt;&gt;` is not equal to `>>>` at all.

Reply:
Either I does not understand something or it does unescape them automatically by calling text()
![45345345](https://user-images.githubusercontent.com/10332458/73124071-dc84cb00-3fa7-11ea-8217-045918160c49.png)

Reply:
I guess Parser.unescapeEntities() is not needed at all?

Reply:
I should probably read the comment on the function next time, because it says that it's unencoded.